src/views/footerView.jsx

```diff
@@ -52,19 +56,48 @@ export const ConferenceFooter = ({ isChatOpen, toggleChat }) => {
   const history = useHistory();
   const params = useParams();
   const pluginRef = useRef(null);
+  const audiopluginRef = useRef(null);
   const isAllowedToPublish = useHMSStore(selectIsAllowedToPublish);
   const permissions = useHMSStore(selectPermissions);
   const [showEndRoomModal, setShowEndRoomModal] = useState(false);
   const [lockRoom, setLockRoom] = useState(false);
   const [showMenu, setShowMenu] = useState(false);
 
+  const noiseSupression = useHMSStore(
+    selectIsLocalAudioPluginPresent("@100mslive/hms-noise-suppression")
+  );
+
+
   const initialModalProps = {
     show: false,
     title: "",
     body: "",
   };
   const [errorModal, setErrorModal] = useState(initialModalProps);
 
+  function createNoiseSupresionPlugin() {
+    if (!audiopluginRef.current) {
+
+      audiopluginRef.current = new HMSNoiseSuppressionPlugin();
+    }
+  }
+
+  async function startAudioPlugin() {
```

Comment:
can we make this - `addNoiseSuppressionPlugin` and `removeNoiseSuppressionPlugin`, similar for vb functions as well to make these more clear for people going through the sample app.